src/main/scala/io/sdkman/changelogs/BTraceMigrations.scala

```diff
@@ -0,0 +1,48 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "027")
+class BTraceMigrations {
```

Comment:
We need the candidate itself to be created, so you would need a changeset for this. Here is a link to the relevant docs:
https://github.com/sdkman/sdkman-db-migrations#adding-a-new-candidate